### PR TITLE
update VID and PID for h87a

### PR DESCRIPTION
**Pull request by hineybush**
## Description

Update PID and VID and vendor of h87a

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/hineybush/h87a/config.h [hunk 1]
```diff
@@ -16,10 +16,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x0000
+#define VENDOR_ID       0x04D8
```
Comment (yanfali) [inline keyboards/hineybush/h87a/config.h]: You are aware that's microchips vendor ID?

Comment (Wilba6582) [reply keyboards/hineybush/h87a/config.h]: In the VIA how-to (coming soon), I am suggesting using two ASCII characters representing the "vendor", in this case, something like "HB" would make sense ;-)

Comment (hineybush) [reply keyboards/hineybush/h87a/config.h]: Yes, I'm aware. I did this:

https://www.microchip.com/usblicensing/

Comment (yanfali) [reply keyboards/hineybush/h87a/config.h]: Ah great, did not know that was an option. Thanks for clarifying

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**